packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -569,35 +569,28 @@ describe("Garbage Collection Tests", () => {
 				defaultGCData.gcNodes[nodes[2]] = [];
 				await garbageCollector.collectGarbage({});
 
-				// Validate that the sweep ready event is logged when GC runs after load.
-				if (expectDeleteLogs) {
-					mockLogger.assertMatch(
-						[{ eventName: deleteEventName, timeout, id: nodes[3] }],
-						"sweep ready event not generated as expected",
-					);
-				} else {
-					mockLogger.assertMatchNone(
-						[{ eventName: deleteEventName }],
-						"Should not have any delete events logged",
-					);
-				}
+				// Since old snapshots get ignored now, we only accept new snapshot formats
+				mockLogger.assertMatchNone(
+					[{ eventName: deleteEventName }],
+					"Should not have any delete events logged",
+				);
 
-				// Validate that all events are logged as expected.
+				// Validate that no events are generated since none of the timeouts have passed
 				garbageCollector.nodeUpdated(nodes[3], "Changed", Date.now(), testPkgPath);
 				garbageCollector.nodeUpdated(nodes[3], "Loaded", Date.now(), testPkgPath);
 				await garbageCollector.collectGarbage({});
-				mockLogger.assertMatch(
+				mockLogger.assertMatchNone(
 					[
 						{ eventName: changedEventName, timeout, id: nodes[3], pkg: eventPkg },
 						{ eventName: loadedEventName, timeout, id: nodes[3], pkg: eventPkg },
 					],
 					"all events not generated as expected",
 				);
 
-				// Add reference from node 2 to node 3 and validate that revived event is logged.
+				// No revived events should be logged as no timeouts should have occurred
 				garbageCollector.addedOutboundReference(nodes[2], nodes[3]);
 				await garbageCollector.collectGarbage({});
-				mockLogger.assertMatch(
+				mockLogger.assertMatchNone(
```

Comment:
It's worth adding an explicit test that validates base snapshot will be undefined for old format GC data. See this test for example - https://github.com/agarwal-navin/FluidFramework/blob/71bd975e81e794f2f035570c75f52da2d0212257/packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts#L846